pdns/dnsdistdist/test-dnsdistbackend_cc.cc

```diff
@@ -179,7 +179,7 @@ BOOST_AUTO_TEST_CASE(test_Lazy)
   BOOST_CHECK_EQUAL(ds.healthCheckRequired(), true);
 
   /* we need maxCheckFailures failed health-checks to go down */
-  for (size_t idx = 0; idx < config.maxCheckFailures - 1; idx++) {
+  for (size_t idx = 0; idx < static_cast<size_t>(config.maxCheckFailures - 1); idx++) {
```

Comment:
this will roll for a while if maxCheckFailures == 0

Reply:
It's good that we explicit set `config.maxCheckFailures = 3;` above, then :)